src/app/interfaces/cloud-sync-task.interface.ts

```diff
@@ -0,0 +1,52 @@
+import { EntityJob } from 'app/interfaces/entity-job.interface';
+import { TransferMode } from 'app/enums/transfer-mode.enum';
+import { Direction } from 'app/enums/direction.enum';
+import { DataProtectionTaskState } from 'app/interfaces/data-protection-task-state.interface';
+import { Schedule } from 'app/interfaces/schedule.interface';
+
+export interface CloudCredential {
+  id: number;
+  name: string;
+  provider: string;
+  attributes: { [key: string]: string };
+}
+
+export interface BwLimit {
+  time: string;
+  bandwidth: number;
+}
+
+export interface CloudSyncTask {
```

Comment:
If any of the new interfaces are coming from API, you can update api-directory file.